src/conversions.js

```diff
@@ -837,3 +835,10 @@ convert.rgb.gray = function (rgb) {
 	const val = (rgb[0] + rgb[1] + rgb[2]) / 3;
 	return [val / 255 * 100];
 };
+
+const models = Object.keys(convert);
```

Comment:
I think _this_ is what's saving some time, _not_ the bundling.

`color-convert` only has 3 files, I don't think that dropping 2 local `require` calls has that much of an effect and probably isn't worth the added complexity.